crates/biome_js_analyze/src/lint/nursery/use_import_extensions.rs

```diff
@@ -82,18 +107,39 @@ declare_rule! {
     }
 }
 
+#[derive(Clone, Debug, Default, Deserializable, Deserialize, Serialize, Eq, PartialEq)]
+#[cfg_attr(feature = "schemars", derive(JsonSchema))]
+#[serde(rename_all = "camelCase", deny_unknown_fields)]
+pub struct UseImportExtensionsOptions {
+    /// A map of custom import extension mappings, where key is inspected file extension
+    /// and value is a pair of `import` extension and `component` import extension
+    pub suggested_extensions: FxHashMap<String, SuggestedExtensionMapping>,
+}
+
+#[derive(Debug, Clone, Default, Deserializable, Deserialize, Serialize, Eq, PartialEq)]
+#[cfg_attr(feature = "schema", derive(schemars::JsonSchema))]
+#[serde(rename_all = "camelCase", deny_unknown_fields)]
+pub struct SuggestedExtensionMapping {
+    /// Extension that should be used for module imports
+    pub import: String,
```

Comment:
I would call this `module`, because that's exactly what it is.

Reply:
Indeed, `module` seems better.